packages/backend/src/index.ts

```diff
@@ -80,16 +82,26 @@ async function main() {
   const graphqlEnv = useHotMemoize(module, () => createEnv('graphql'));
   const appEnv = useHotMemoize(module, () => createEnv('app'));
 
+  passport.use(
```

Comment:
I'm thinking it's best to not make any modifications to the backend for now, and simply make the middleware available for experimental use.

This applies the middleware very broadly, almost using it as an access control mechanism for the entire app. I'd rather stick to trying to figure out the more narrow use-case of how to allow certain plugins to use the authenticated user to control access of some resources.

App-wide access restrictions would have to be cookie-based along with some dedicated frontend code to handle the initial login.

Reply:
Agreed in as much as these changes to the example-backend should be removed before merge. My intent was to exemplify usage to make review easier to discuss, and then if we reach some decision move this part to documentation, as one or more examples on how adopters can choose to apply authorization depending on their environment. Since this type of authorization requires clients to pass the token in the request, it is also wise to allow authorization to be applied to individual plugins separately until they have all been assimilated.

Not sure I understand the cookie requirement (this code works as far as I can test), as it is only applied to api routes?